### PR TITLE
refine CLI help messages

### PR DESCRIPTION
**Pull request by changweige**
Make them more suggestive

Signed-off-by: Changwei Ge <gechangwei@bytedance.com>

### CODE REVIEW

**Comment by sctb512** [comment]
Others, LGTM

File: cmd/containerd-nydus-grpc/pkg/command/flags.go [hunk 1]
```diff
@@ -73,38 +71,35 @@ func buildFlags(args *Args) []cli.Flag {
...
 			Name:        "log-to-stdout",
-			Usage:       "log messages to standard out rather than files.",
+			Usage:       "print log messages to STDOUT",
 			Destination: &args.LogToStdout,
 		},
 		&cli.StringFlag{
 			Name:        "nydus-image",
-			Value:       "",
-			Usage:       "set `PATH` to the nydus-image binary, default to lookup nydus-image in $PATH",
+			Usage:       "`nydus-image` binary path, it will be searched if this option is not provided",
```
[the remark this answers is not in the document]
Comment (changweige) [reply cmd/containerd-nydus-grpc/pkg/command/flags.go]: fixed

**Review by sctb512 — APPROVED** [review]

File: cmd/containerd-nydus-grpc/pkg/command/flags.go [hunk 2]
```diff
@@ -73,38 +71,35 @@ func buildFlags(args *Args) []cli.Flag {
...
 			Name:        "nydus-image",
-			Value:       "",
-			Usage:       "set `PATH` to the nydus-image binary, default to lookup nydus-image in $PATH",
+			Usage:       "`nydus-image` binary path, it will be searched from $PATH if this option is not provided",
 			Destination: &args.NydusImagePath,
 		},
 		&cli.StringFlag{
 			Name:        "nydusd",
-			Value:       "",
-			Aliases:     []string{"nydusd-path"},
```
Comment (imeoer) [inline cmd/containerd-nydus-grpc/pkg/command/flags.go:106]: Can we still keep the alias?

Comment (changweige) [reply cmd/containerd-nydus-grpc/pkg/command/flags.go:106]: Better not I suppose, thus to keep the consistency everywhere and simply the flows on how to use nydus

**Review by imeoer — APPROVED** [review]

**Pull request merged**